README.md

```diff
@@ -90,19 +90,19 @@ apply plugin: 'com.github.triplet.play'
 
 #### Snapshot builds
 
-If you're prepared to cut yourself on the bleeding edge of this plugin, snapshot builds are
-available through JitPack:
+If you're prepared to cut yourself on the bleeding edge of this plugin, snapshot builds are available in [Sonatype's
+`snapshots` repository](https://oss.sonatype.org/content/repositories/snapshots/):
```

Comment:
Hmmm, still over 140 chars. I would just do this:
```md
If you're prepared to cut yourself on the bleeding edge of this plugin, snapshot builds are available
in [Sonatype's `snapshots` repository](https://oss.sonatype.org/content/repositories/snapshots/):
```

Reply:
Actually, that's 117 chars :) There are at least 4 other lines that have the same length.

My IDE prints a vertical line at 120 chars, that's my reference for manual line wrapping.

Reply:
Ok then, SGTM 👍